source/monthly-planner.html

```diff
@@ -5,39 +5,63 @@
         <link rel="stylesheet" href="styles/monthly-planner-style.css" />
-        <script src="monthly-planner.js" type="module"></script>
         <script src="monthly-parser.js" type="module"></script>   
     </head>
     <body>
         <div class="taskbar-wrapper">
-            <h1>Backlog</h1>
-            <button id="switch-to-weekly">Switch to Weekly View</button>
-            <hr />
-            <h2>Hello, %name%!</h2>
-            <h3 id="task-count">You have 0 tasks due for May 2024.</h3>
-            <form id="task-form">
+            <div class="taskbar-navigator">
+                <button id="options" class="taskbar-navigator-button">
+                    Options
+                </button>
+                <h1>Task List</h1>
+                <button id="switch-to-weekly" class="taskbar-navigator-button">
+                    Switch to Weekly View
+                </button>
+            </div>
+            <div class="taskbar-search-bar">
                 <input
+                    id="taskbar-option-search-bar"
                     type="text"
-                    id="new-task"
-                    placeholder="Enter new task"
-                    required
+                    placeholder="Enter task to search..."
                 />
-                <button type="submit">Add Task</button>
-            </form>
-            <ul id="task-list"></ul>
+            </div>
+            <div class="taskbar-options" hidden>
+                <label>Order by:</label>
+                <select id="taskbar-option-sort-order">
+                    <option value="IMPORTANCE" selected>Importance</option>
+                    <option value="DUE_DATE">Due Date</option>
+                </select>
+                <hr />
+                <label>Completed:</label>
+                <input type="checkbox" id="taskbar-option-completed" checked />
+                <hr />
+                <label>Between:</label>
+                <input type="date" id="taskbar-option-start-date" />
+                ~
+                <input type="date" id="taskbar-option-end-date" />
+            </div>
+            <h4 id="taskbar-search-status" hidden></h4>
+            <ul id="taskbar-list"></ul>
         </div>
         <div class="calendar-wrapper">
             <div class="navigator">
                 <h1 class="month-title">May 2024</h1>
                 <div class="navigator-buttons">
-                    <button id="prevButton" class="navigator-button">
+                    <button
+                        id="add-button"
+                        class="navigator-button edit-button"
+                    >
+                        +
+                    </button>
+                    <button id="prev-button" class="navigator-button">
                         &#x3008;
                     </button>
-                    <button id="curButton" class="navigator-button">
+                    <button id="cur-button" class="navigator-button">
```

Comment:
good standardizing of these ids, we probably need to make more improvements like this across the whole repo